src/sentry/static/sentry/app/views/organizationGroupDetails/reprocessingProgress.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import ProgressBar from 'app/components/progressBar';
+import {t, tct, tn} from 'app/locale';
+import space from 'app/styles/space';
+
+type Props = {
+  totalEvents: number;
+  pendingEvents: number;
+};
+
+function ReprocessingProgress({totalEvents, pendingEvents}: Props) {
+  const remainingEventsToReprocess = totalEvents - pendingEvents;
+  const remainingEventsToReprocessPercent =
+    (remainingEventsToReprocess / totalEvents) * 100;
+
+  return (
+    <Wrapper>
+      <Inner>
+        <Header>
+          <Title>{t('Reprocessing\u2026')}</Title>
+          {t(
+            'Once the events in this issue have been reprocessed, you’ll be able to make changes and view any new issues that might be created.'
+          )}
+        </Header>
+        <Content>
+          <ProgressBar value={remainingEventsToReprocessPercent} variant="large" />
+          {tct('[remainingEventsToReprocess]/[totalEvents] [event] reprocessed', {
+            remainingEventsToReprocess,
+            totalEvents,
+            event: tn('event', 'events', totalEvents),
+          })}
+        </Content>
+      </Inner>
+    </Wrapper>
+  );
+}
+export default ReprocessingProgress;
+
+const Wrapper = styled('div')`
+  margin: 30px 40px;
+  flex: 1;
+  text-align: center;
+
+  @media (min-width: ${p => p.theme.breakpoints[0]}) {
+    margin: 40px;
+  }
+`;
+
+const Content = styled('div')`
+  color: ${p => p.theme.gray300};
+  font-size: ${p => p.theme.fontSizeMedium};
+  display: grid;
+  grid-gap: ${space(1.5)};
+  max-width: 402px;
+  width: 100%;
+`;
+
+const Inner = styled('div')`
+  display: grid;
+  grid-gap: ${space(3)};
+  justify-items: center;
+`;
+
+const Header = styled('div')`
+  display: grid;
+  grid-gap: ${space(1)};
+  color: ${p => p.theme.gray500};
+  max-width: 557px;
+`;
+
+const Title = styled('div')`
```

Comment:
Are we able to use an `h*` element for headings?

Reply:
I wasn't sure which one to use here, so I preferred to leave it as a div

Reply:
h3 is a good choice for 'not the primary header' on a page.